src/commands/generate/schemaTypes.ts

```diff
@@ -681,7 +681,7 @@ const fillInUnresolvedSchemas = (specs: SchemaSpec[]): SchemaSpec[] => {
           },
         };
         shell.echo(
-          chalk.yellow(`WARNING: Referenced Schema '${unresolved.path}' is unresolved. Falling back to 'unknown' type.`)
+          chalk.yellow(`WARNING: Schema '${unresolved.path}' referenced from '${spec.contextName}' is unresolved. Falling back to 'unknown' type fro '${unresolved.path}'.`)
```

Comment:
What does a type fro even look like? Is that related to an afro? :trollface: 

```suggestion
          chalk.yellow(`WARNING: Schema '${unresolved.path}' referenced from '${spec.contextName}' is unresolved. Falling back to 'unknown' type for '${unresolved.path}'.`)
```